Data/Vector.hs

```diff
@@ -1519,6 +1520,12 @@ maximumBy :: (a -> a -> Ordering) -> Vector a -> a
 {-# INLINE maximumBy #-}
 maximumBy = G.maximumBy
 
+-- | /O(n)/ Yield the maximum element of the vector according to the given
+-- comparison function. The vector may not be empty.
```

Comment:
While I understand what you mean, it is misleading to call `a -> b` a "comparison function". It does not compare anything, right? I'd say "Yield an element, at which the given function takes the maximum values amongst all elements of the vector" or similar. 

Also it would be nice to mention how ties are resolved (e. g., arbitrary?).

Reply:
> Also it would be nice to mention how ties are resolved (e. g., arbitrary? the left-most element? the right-most element?).

I forgot to add this in 536d57baa263c950d2e7854d40deb6ad1162d4a7, I will include it in a second.